commons/src/main/java/fr/free/nrw/commons/upload/ShareActivity.java

```diff
@@ -175,13 +175,24 @@ public void onCreate(Bundle savedInstanceState) {
         mediaUriString = mediaUri.toString();
         Log.d("Image", "Uri: " + mediaUriString);
 
+        //convert image Uri to file path
         FilePathConverter uriObj = new FilePathConverter(this, mediaUri);
         String filePath = uriObj.getFilePath();
 
+        //extract the coordinates of image in decimal degrees
         GPSExtractor imageObj = new GPSExtractor(filePath);
         String coords = imageObj.getCoords();
         Log.d("Image", "Coords of image: " + coords);
 
+        //build URL with image coords for MediaWiki API calls
+        String apiUrl = UrlBuilder.buildUrl(coords);
+        Log.d("Image", "URL: " + apiUrl);
```

Comment:
:+1: Good call to log the URLs requested. It's good to verify what's encoded / not encoded when working.